api-reference/beta/api/group_delta.md

```diff
@@ -18,6 +18,36 @@ To begin tracking changes, you make a request including the delta function on th
 ```http
 GET /groups/delta
 ```
+
+### Query parameters
+
+Tracking changes in messages incurs a round of one or more **delta** function calls. If you use any query parameter 
```

Comment:
You probably want to customize the first part of the first sentence here as "Tracking changes in _groups_ incurs..."